src/schema/artwork/index.js

```diff
@@ -310,6 +310,10 @@ export const artworkFields = () => {
         return Image.resolve(size ? _.take(sorted, size) : sorted)
       },
     },
+    inventory_id: {
```

Comment:
Sorry for only catching this now, but shouldn't this be `inventoryId`?

Reply:
Yes, thanks for catching it. I will PR that.